lib/pliny/error_reporter.rb

```diff
@@ -14,29 +13,35 @@ def self.notify(exception, context: {}, rack_env: {})
     end
   end
 
-  class RollbarReporter
-    include ::Rollbar::ExceptionReporter
-    include ::Rollbar::RequestDataExtractor
+  begin
+    require 'rollbar/exception_reporter'
+    require 'rollbar/request_data_extractor'
 
-    def notify(exception, context:, rack_env:)
-      Rollbar.reset_notifier!
-      scope = fetch_scope(context: context, rack_env: rack_env)
-      Rollbar.scoped(scope) do
-        report_exception_to_rollbar(rack_env, exception)
+    class RollbarReporter
```

Comment:
I'm not a big fan of having this in the main error_reported file.
How about having `lib/pliny/error_reporters/rollbar`?